lesson_15/tests/test_polyon.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from lesson_15.lesson_15_mihaela_vaida import polygon
+
+
+class TestPolygon(unittest.TestCase):
+    def test_polygon_object_len_of_sides(self):
+        sides = 4, 3, 4, 5
```

Comment:
this test is good, but will never fail even if we change the code. You need to test the sides from test_polygon not the ones from your preparation variable `sides`.

You can also uncomment the cod that is rising an exception because is a good implementation. Take a look at this already build assert from unittest that can test if the __init__ is rising an exception https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises

Reply:
modified !